src/components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../css/Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <footer className="pageFooter">
```

Comment:
Kabob case css class names, please

```suggestion
      <footer className="page-footer">
```